build/import/NuGet.props

```diff
@@ -1,10 +0,0 @@
-<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information. -->
```

Comment:
We don't use this functionality (change the packages based on environment variable). `NuGetPackageRoot` already exists as part of the build normally.